nix/pkgs/extensions/cargo-project.nix

```diff
@@ -133,7 +133,11 @@ let
         export OPENSSL_LIB_DIR=${static_ssl.out}/lib
         export OPENSSL_INCLUDE_DIR=${static_ssl.dev}/include
       '';
-      ${if static then "RUSTFLAGS" else null} = [ "-C" "target-feature=+crt-static" ];
+      RUSTFLAGS =
+        if static then
+          [ "-C" "target-feature=+crt-static" ]
+        else
+          [ "-C" "strip=debuginfo" "-C" "debuginfo=0" ];
```

Comment:
removal of debug should be opt-in

Reply:
Alright

Reply:
I'm inputting the RUSTFLAGS using an env.
```
RUSTFLAGS="-C debuginfo=0 -C strip=debuginfo"
```